cli/src/commands/extensions/permissions.rs

```diff
@@ -228,6 +231,7 @@ impl Permissions {
             env: self.env.subset_of(&other.env).map_err(err_ctx("env"))?,
             run: self.run.subset_of(&other.run).map_err(err_ctx("run"))?,
             net: self.net.subset_of(&other.net).map_err(err_ctx("net"))?,
+            unsandboxed_run: Permission::default(),
```

Comment:
Can we put a comment here clarifying the reason of this being different from the above? I _assume_ that it's because it has nothing to do with `birdcage` and doesn't carry over in the sandbox layering, but I may be missing something and it took a bit of time for me to get to that conclusion.

Reply:
Yeah `unsandboxed_run` is **only** for `Deno.run`, so obviously we don't want it anywhere near sandboxed run.